repos/system_upgrade/el8toel9/actors/sssdfacts/tests/unit_test_sssdfacts_8to9.py

```diff
@@ -0,0 +1,168 @@
+import textwrap
+
+from six import StringIO
+
+from leapp.libraries.actor.sssdfacts8to9 import SSSDFactsLibrary
+from leapp.libraries.common import utils
+
+
+def get_config(content):
+    parser = utils.parse_config(StringIO(textwrap.dedent(content)))
```

Comment:
Nit: can return straightahead, no need for variable assignment

Reply:
Done.